regulations/static/regulations/css/less/module/comment.less

```diff
@@ -4,12 +4,20 @@ comment.less
 comment.less contains styles related to commenting on sections
 */
 
+@comment_blue: #4A90E2;
+@comment_blue_dark: #00629f;
+@comment_blue_light: #BDDCFD;
+@comment_item_blue: #E2F3F8;
+@comment_item_gray: #AEB0B5;
+@comment_input_border: #E5E5E5;
```

Comment:
This is much better than having the colors throughout, but we should aim to reuse variables which already exist in our pallet. We don't have a semantic pallet at the moment, but you could imagine this instead saying something like:
```less
@comment_active = @secondary_accent;
```